packages/admin/admin/src/dataGrid/GridColDef.ts

```diff
@@ -0,0 +1,11 @@
+import { GridColDef as MuiGridColDef, GridValidRowModel } from "@mui/x-data-grid";
+
+export type GridColView = "default" | "compact";
+
+export interface GridColDef<R extends GridValidRowModel = any, V = any, F = V> extends MuiGridColDef<R, V, F> {
+    /**
+     * Media query to define when the column is visible.
+     * Requires DataGridPro or DataGridPremium.
+     */
+    visible?: string;
```

Comment:
Can we find a better name that doesn't require this comment to understand it?

Reply:
`visibleMediaQuery`?

Reply:
> Requires DataGridPro or DataGridPremium.

Should we keep this part of the comment?

Reply:
IMO `visible: theme.breakpoints.down("md")` was more "beautiful" than `visibleMediaQuery: theme.breakpoints.down("md")`. Could we achieve having no comment (is this really that bad?) by using a type alias?

```ts
type MediaQueryFromTheme = string; // Or "MediaQuery"

...

visible?: MediaQueryFromTheme;
```

Reply:
> IMO visible: theme.breakpoints.down("md") was more "beautiful" than visibleMediaQuery: theme.breakpoints.down("md")

True, but it's also less descriptive...

> Could we achieve having no comment (is this really that bad?) by using a type alias?
> 
> ```ts
> type MediaQueryFromTheme = string; // Or "MediaQuery"
> 
> ...
> 
> visible?: MediaQueryFromTheme;
> ```

Unfortunately, when hovering in the IDE, the name of the type is not shown, if that's what you intended. 
Only the underlying type is shown. You'd have to navigate into the type to see the name. 

| VSCode | PhpStorm |
| ---------- | --- |
| <img width="588" alt="VSCode" src="https://github.com/vivid-planet/comet/assets/6264317/6b975f74-8a6b-4ea3-9518-f665798f084f"> | <img width="470" alt="PhpStorm" src="https://github.com/vivid-planet/comet/assets/6264317/f0b35719-dc13-4772-ba88-165fefe90097"> |

Reply:
I think `visible` is fine. 
Having the comment is helpful but not necessary imo.